src/bindgen/ir/cfg.rs

```diff
@@ -286,21 +375,33 @@ impl CfgWrite for Option<Cfg> {
                 return;
             }
 
-            out.write("#if ");
-            cfg.write_condition(config, out);
-            out.new_line();
+            let mut unmatched = vec![];
+            if let Some(cfg) = cfg.matched_defines(config, &mut unmatched) {
```

Comment:
Do we still need the `if !cfg.has_defines(config)` line above, if we have this?

Reply:
We don't. Fixed.